### DISCUSSION TITLE
Can't install oh-my-zsh on unraid 6.11.0

### DISCUSSION THREAD

**Discussion by turkeyphant** [ohmyzsh/ohmyzsh#11976] [Help] [answered]
### Describe the bug

Running install script doesn't work any more.

### Steps to reproduce

1. Run `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)" "" --unattended`
2. Quits with error `Zsh is not installed. Please install zsh first.`

### Expected behavior

Installs zsh as previously

### OS / Linux distribution

Unraid 6.11.0

### Zsh version

latest

### Terminal emulator

?

**Comment by rilysh** [comment]
You need to install zsh (shell) first. Check their docs regardless of whatever package manager they're using.

**Comment by lnxd** [comment, marked as answer]
@rilysh is spot on. To install zsh there are two paths that I'd suggest as Unraid doesn't come with a package manager.

Option 1. 
Follow the steps provided [here](https://forums.unraid.net/topic/35866-unraid-6-nerdpack-cli-tools-iftop-iotop-screen-kbd-etc/?do=findComment&comment=1159381) which state:
```
To replicate the functionality (unsupported):
- Go to https://slackware.pkgs.org/15.0/slackware-x86_64/ which 
lists packages for Slackware 15 Unraid is based on
Search for the packages you want
- Download the txz files for them, and put them on the flash drive 
in /extra (/boot/extra on a running system), that will cause them to 
auto-install on boot (create the folder if there isn't one)
- To be able to use them without a reboot use unraid CLI to navigate 
where you put the packages and run installpkg <filename>
```
Option 2. 
Slightly more advanced option would be to manually install @ich777's un-get plugin available [here](https://github.com/ich777/un-get). This is more advanced but if you install packages regularly, will make things a lot easier. 

Note that while installing a package such as zsh is unlikely to cause any problems, it's not supported by Lime, and there are a lot of plugins available via slackware that will irreparably damage your Unraid installation: Unraid isn't a standard Linux distro, and relies on a lot of proprietary customisation. Please ensure you back up your flash drive first, and be careful what you install.

Best to get support for this issue over on the [Unraid forums](https://forums.unraid.net/) though.
